plugins/source/snyk/resources/services/organization/provisions.go

```diff
@@ -0,0 +1,57 @@
+package organization
+
+import (
+	"context"
+
+	"github.com/cloudquery/cloudquery/plugins/source/snyk/client"
+	"github.com/cloudquery/plugin-sdk/v2/schema"
+	"github.com/cloudquery/plugin-sdk/v2/transformers"
+	"github.com/pavel-snyk/snyk-sdk-go/snyk"
+)
+
+func pendingUserProvisions() *schema.Table {
+	return &schema.Table{
+		Name:        "snyk_organization_provisions",
+		Description: `https://snyk.docs.apiary.io/#reference/organizations/provision-user/list-pending-user-provisions`,
+		Resolver:    fetchPendingProvisions,
+		Multiplex:   client.ByOrganization,
+		Transform: transformers.TransformWithStruct(&snyk.PendingProvision{},
+			transformers.WithPrimaryKeys("Email", "Created")),
+		Columns: []schema.Column{
+			{
+				Name: "organization_id",
```

Comment:
create common org id col in client pkg, use it here to retrieve the current org id?